### DISCUSSION TITLE
Comet version of SwiftUI's ForEach?

### DISCUSSION THREAD

**Discussion by GeoffArmstrong** [dotnet/Comet#258] [Q&A]
Is there a Comet version of SwiftUI's ForEach view? What is the best way to handle non-virtualized repeated controls? Is there an ID concept to allow the MVU updater to identify exactly what has changed after a list change?

**Comment by tcsaddul** [comment]
Maybe this Sample? 
https://github.com/dotnet/Comet/blob/dev/sample/Comet.Sample/Views/ListViewSample1.cs
